hardware/qcom/Android.mk

```diff
@@ -23,9 +23,9 @@ audio-hal := hardware/qcom/audio
 bt-hal := hardware/qcom/bt/sdm845
 gps-hal := hardware/qcom/gps/sdm845
 display-hal := hardware/qcom/display/sde
-QCOM_MEDIA_ROOT := hardware/qcom/media/sdm845
+QCOM_MEDIA_ROOT := hardware/qcom/media/msm8998
 OMX_VIDEO_PATH := mm-video-v4l2
-media-hal := hardware/qcom/media/sdm845
+media-hal := hardware/qcom/media/msm8998
```

Comment:
media-hal := ${QCOM_MEDIA_ROOT} please :)

Reply:
@oshmoun good idea doing in new commit

Reply:
please squash https://github.com/sonyxperiadev/device-sony-common/pull/505/commits/1547cfbe8976a0dd65e6e53584057572c23afc03